src/test/java/com/ibm/cloud/sdk/core/util/discriminator/AcceptPayloadWithoutDiscriminatorMapping.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * (C) Copyright IBM Corp. 2021.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.cloud.sdk.core.util.discriminator;
+
+public class AcceptPayloadWithoutDiscriminatorMapping extends StatusPayloadWithoutDiscriminatorMapping {
+
+}
```

Comment:
I am not quite sure about the best style here, but I think the newline shouldn't be here.
```suggestion
public class AcceptPayloadWithoutDiscriminatorMapping extends StatusPayloadWithoutDiscriminatorMapping {
}
```

Reply:
I would go without newline, but both version is ok (according to the Google style guide), so it's up to you which one you choose.
```suggestion
public class AcceptPayloadWithoutDiscriminatorMapping extends StatusPayloadWithoutDiscriminatorMapping {}
```